ledgerblue/recoverBackup.py

```diff
@@ -0,0 +1,159 @@
+from ledgerblue.comm import getDongle
+from ledgerblue.hexLoader import HexLoader
+from ledgerblue.recoverMutualAuth import recoverMutualAuth, SCPv3, recoverValidate
+from ledgerblue.recoverUtil import Recover
+from hashlib import sha256
+from Crypto.Cipher import AES
+import binascii
+import argparse
+import json
+import gnupg
+import os
+
+
+def get_argparser():
+    parser = argparse.ArgumentParser(description="Backup 3 shares of a seed.")
+    parser.add_argument("--targetId", help="The device's target ID (default is Nano X)", type=auto_int)
+    parser.add_argument("--rootPrivateKey", help="The private key of the Certificate Authority")
+    parser.add_argument("--issuerPublicKey", help="The public key of the Issuer (used to verify the certificate of "
+                                                  "the device)")
+    parser.add_argument("--nWords", help="The number of words in the mnemonic (default is 24 words)", type=auto_int)
```

Comment:
Any chance we can change this to `numberOfWords`? `nWords` has a bad ring to it 😅